### PR TITLE
Edits to Guide Unit 1

### PR DESCRIPTION
**Pull request by patrickscottd**
These are my first edits
Patrick

### CODE REVIEW

File: web/help/GuideUnit1.md [hunk 1]
```diff
@@ -593,21 +619,21 @@ more hints, expand the sections below for an example and explanation.
...
 had some big limitations.  First, they were black and white.  Colors would
-make them much more exciting.  And second, all the shapes were in the
+make them much more exciting.  Second, all the shapes were in the
 center of the screen.  You can fix both of these problems using
 *transformations*.
 
 A **transformation** is a function that receives a picture called the
-preimage, and a description of something to change about it, and
-produces a new picture that is like the preimage except for that change.
+preimage, with a description of something to change about it, and then
```
Comment (cdsmith) [inline web/help/GuideUnit1.md:635]: I used "and" deliberately there, because the preimage and the changes are both arguments side-by-side.  Students often fail to understand that the picture is an argument.  Maybe rewording would make this more clear.

For instance:

```
A **transformation** is a function whose arguments are:

1. a picture, called the preimage and
2. some changes to be made to the preimage.

The result of the transformation is a new picture that is like the preimage, except for that change.
```

File: web/help/GuideUnit1.md [hunk 2]
```diff
@@ -625,9 +651,9 @@ Coloring
 --------
 
 The first transformation you will use is coloring.  The `colored` function
-changes the color of a picture. This function expects two arguments: the
-preimage, and a new color.  The colors of the preimage don't matter at
-all; only the shapes involved.  The result of the `colored` function is a new
+changes the color of a picture. This function expects two arguments: 1) the
+preimage and 2) a new color.  The colors of the preimage don't matter at
+all, only the shapes involved.  The result of the `colored` function is a new
```
Comment (cdsmith) [inline web/help/GuideUnit1.md:656]: I don't think either a comma or semicolon is correct here, actually.  A dash should work.

File: web/help/GuideUnit1.md [hunk 3]
```diff
@@ -636,62 +662,66 @@ redWheel = colored(wheel, red)
...
 
 !!! collapsible: What does the picture named `wheel` look like?
-    It's a filled-in circle, of radius 4, drawn in **black**.
+    It's a filled-in circle of radius 4, drawn in **black**.
 
     <div align="center"><iframe src="https://code.world/run.html?mode=codeworld&dhash=DrYNeySBqPKubPkzT48dTEA" width=250 height=250 style="border: none;"></iframe></div>
 
     Surprised?  If so, this is a good chance to review what a
-    transformation does--and what it doesn't do.  The equation
+    transformation does do and, also, what it doesn't do.  The equation
```
Comment (cdsmith) [inline web/help/GuideUnit1.md:674]: How about "what a transformation does and doesn't do." ?

File: web/help/GuideUnit1.md [hunk 4]
```diff
@@ -636,62 +662,66 @@ redWheel = colored(wheel, red)
...
-    you expected.  But that *doesn't* change the meaning of the other
+    you expected, but that *doesn't* change the meaning of the other
     variable, `wheel`.  Remember that `redWheel` and `wheel` are
     different pictures.
 
     The final program draws a wheel in red, because you defined the
     **program** to be a drawing of `redWheel`.
 
-    Can you guess what the result of this code is?
+Can you guess what the result of this code is?
```
Comment (cdsmith) [inline web/help/GuideUnit1.md:685]: Needs an indent to avoid breaking the box.

File: web/help/GuideUnit1.md [hunk 5]
```diff
@@ -636,62 +662,66 @@ redWheel = colored(wheel, red)
...
-    because `program` is defined to be a drawing of `wheel`, which
-    is still black.
+    If you ran this, you would see a black circle. `redWheel` is still
+    defined to be a red circle, but that doesn't change anything. 
+    Why?
+    It is because `program` is defined to be a drawing of `wheel`, which
+    is still black. The original input was 'wheel` and 'wheel` is defined 
+    as a `solidCircle` of radius 4 and the call for the input `wheel` in the 
+    first line `program` overrides the call for the input `wheel` in the second 
+    line `redWheel`.
```
Comment (cdsmith) [inline web/help/GuideUnit1.md:700]: I wouldn't say that anything "overrides" anything.

The second line means exactly what it says: The variable `redWheel` is a picture just like `wheel` but drawn in black.  It The point here is that line 2 doesn't change what `wheel` is.  It just defines a new picture.

File: web/help/GuideUnit1.md [hunk 6]
```diff
@@ -636,62 +662,66 @@ redWheel = colored(wheel, red)
...
 
 When you use `colored` in a program, a *whole expression* like
 `colored(pic, blue)` describes a picture, so you can use an expression
 like this in all the same places you could use any other description of
-a picture.  For example, you might define `nametag` like this.
+a picture.  
+
+For example, you might define `nametag` like this
```
Comment (cdsmith) [inline web/help/GuideUnit1.md:707]: punctuation

File: web/help/GuideUnit1.md [hunk 7]
```diff
@@ -636,62 +662,66 @@ redWheel = colored(wheel, red)
...
 
 ~~~~~ . clickable
 program = drawingOf(nametag)
-
```
Comment (cdsmith) [inline web/help/GuideUnit1.md:679]: Any reason to remove these blank lines?  I'd like to encourage students to space out their code so it's easier to find sections at a glance.

File: web/help/GuideUnit1.md [hunk 8]
```diff
@@ -636,62 +662,66 @@ redWheel = colored(wheel, red)
...
 outerBorder = thickRectangle(15, 15, 1)
 innerBorder = thickCircle(6, 1)
 name        = lettering("Winona")
 ~~~~~
 
 Since `colored(outerBorder, blue)` is an expression that describes a
 picture, you can combine it with other pictures using **`&`**, just
-like you could with `circle(5)`, `rectangle(3, 4)`, or any other
+like you could with `circle(5)`, `rectangle(3, 4)` or any other
```
Comment (cdsmith) [inline web/help/GuideUnit1.md:721]: please keep the Oxford comma

File: web/help/GuideUnit1.md [hunk 9]
```diff
@@ -636,62 +662,66 @@ redWheel = colored(wheel, red)
...
 picture expression.
 
-Here is a table of colors you can use:
+Here is a table of colors you can use
```
Comment (cdsmith) [inline web/help/GuideUnit1.md:724]: punctuation

File: web/help/GuideUnit1.md [hunk 10]
```diff
@@ -716,60 +746,65 @@ Here is a table of colors you can use:
...
 -------------------
 
 The second transformation you can use in your programs is translation.  The
 `translated` function shifts a picture up, down, left, or right.  Its inputs
-are the preimage, and two distances to move the picture.  The result of
-`translated` is a new picture with the same content as the preimage, but
-shifted either horizontally (that is, left or right) or vertically (that is,
-up or down) or both.
+are the preimage and two numbers, which are distances that we want
+to move the picture.  The result of `translated` transformation is a new picture
```
Comment (cdsmith) [inline web/help/GuideUnit1.md:758]: Missing an article?  "The result of **the** `translated` transformation"

File: web/help/GuideUnit1.md [hunk 11]
```diff
@@ -716,60 +746,65 @@ Here is a table of colors you can use:
...
+A useful tool for finding these numbers is the **coordinate plane**.
 
 ![](coordinate-plane.png width=50%)
 
 !!! Tip: Get your own coordinate plane
     The coordinate plane is a picture that your computer knows about in
-    CodeWorld.  To make your own coordinate plane, use this code:
+    CodeWorld.  
+    
+To make your own coordinate plane, use this code
```
Comment (cdsmith) [inline web/help/GuideUnit1.md:773]: Needs to be indented, so the box isn't broken.

File: web/help/GuideUnit1.md [hunk 12]
```diff
@@ -716,60 +746,65 @@ Here is a table of colors you can use:
...
 
     ~~~~~ . clickable
     program = drawingOf(coordinatePlane)
     ~~~~~
 
-This tool combines of two number lines.
+This tool is a combination of two number lines:
+
+1. The first number line is *horizontal*, like the *x-axis*, with positive numbers
```
Comment (cdsmith) [inline web/help/GuideUnit1.md:781]: Saying "like the x axis" is weird, since it is literally the x axis.  What about this?

```
This tool is a combination of two number lines:

1. The first number line is called the *x* *axis*.  It is *horizontal*, with positive numbers
   to the right and negative numbers to the left.
2. The second number line is called the *y* *axis*.  It is *vertical*, with positive numbers
   on the top and negative numbers on the bottom.
```

File: web/help/GuideUnit1.md [hunk 13]
```diff
@@ -716,60 +746,65 @@ Here is a table of colors you can use:
...
+The basic shapes you have learned are drawn with their center at the *origin* --
 the point (0, 0) -- on the coordinate plane, so you should measure your x
 and y distances from there.  As you define your own pictures, it's a good
-idea to continue doing the same thing.
+idea to continue this practice.
 
-For example, if you wanted a circle representing the sun in the top left
-corner of the screen.  First, you could look at the horizontal number line,
+For example, suppose you wanted a circle representing the sun in the top left
+corner of the screen.  First, you could look at the horizontal number line
```
Comment (cdsmith) [inline web/help/GuideUnit1.md:796]: Assuming the change above, you'd want to say "look at the x axis"

File: web/help/GuideUnit1.md [hunk 14]
```diff
@@ -716,60 +746,65 @@ Here is a table of colors you can use:
...
-
-The expression describing a picture of the sun in the right place is now
-`translated(sun, -5, 7)`.  The first argument is the preimage, which you
-would define elsewhere in the program.  The second and third arguments are
-the distances to move the shape in your new picture, with horizontal first,
-then vertical.
+pick x = -5, which is five units moving left from the origin on the screen.  
+Next, you could look at the vertical number line and see that positive numbers are 
+used to move an object up.  You might pick 7, which is most of the way 
+to the top of the screen. This should give you a proper placement for the sun in your image.
```
Comment (cdsmith) [inline web/help/GuideUnit1.md:801]: I am worried a little but about this paragraph.  The idea we want to get across here is that `translated(sun, -5, 7)` means to shift the sun 5 units to the left, and 7 units up.  It would be WRONG if students thought that `translated(sun, -5, 7)` means to move the sun to the point (-5, 7).

To see the difference, you could consider this code:

    a = translated(circle(5), 2, 2)
    b = translated(a, -2, -2)

Students should be able to reason that `b` is a picture of a circle at the origin, not at (-2, -2).  (Here's why: `b` starts with the preimage `a`, which is a circle centered at (2, 2), and then shifts it 2 units left and 2 units down.  That moves it back to the origin.)

Some students will expect that `b` moves the circle **to** (-2, -2), and that it shouldn't matter where it was before it was moved.  That is, they will mistakenly believe that translation is absolute, instead of relative.  I don't know how to address that misconception here without getting into nesting or combining transformations.  But at the very least, we could make sure this isn't actively giving students the wrong idea.

File: web/help/GuideUnit1.md [hunk 15]
```diff
@@ -1,8 +1,8 @@
-Getting started
+Getting Started
```
[the remark this answers is not in the document]
Comment (patrickscottd) [reply web/help/GuideUnit1.md:1]: I like the capitalization, but if it's not worth changing, I totally understand. I like the formality and symmetry, but that is a personal preference.

File: web/help/GuideUnit1.md [hunk 16]
```diff
@@ -16,45 +16,45 @@ CodeWorld logo in the *canvas* on the right side of the screen.
...
 
-Here's the code you just wrote, and what its parts mean.
+Here's the code you just wrote and what its parts mean
 
     | `program`  | `=` | `drawingOf(` | `codeWorldLogo`     | `)` |
     |------------|-----|--------------|---------------------|-----|
     | My program | is  | a drawing of | the CodeWorld logo. |     |
 
 * `program` is the **variable** that you're defining. A variable is a name for
-  something.  In most math, variables are just one letter long, and they stand
```
[the remark this answers is not in the document]
Comment (patrickscottd) [reply web/help/GuideUnit1.md:50]: I seem to make that mistake everywhere. My Word program never points it out, and I rely on it for guidance.

File: web/help/GuideUnit1.md [hunk 17]
```diff
@@ -16,45 +16,45 @@ CodeWorld logo in the *canvas* on the right side of the screen.
...
+  something.  In most math, variables are just one letter long and they stand
   for numbers.  In CodeWorld, though, variables can name many different types of
-  values: numbers, pictures, colors, text, and even whole programs.  Because
```
[the remark this answers is not in the document]
Comment (patrickscottd) [reply web/help/GuideUnit1.md:52]: I read about them just today. I had no idea. I do not remember anything like that from English class, but it's been a really, really long time since I've had English. I've actually taken more German since my last English class and I might have things stuck in my head from it, who knows.

File: web/help/GuideUnit1.md [hunk 18]
```diff
@@ -1,8 +1,8 @@
-Getting started
+Getting Started
 ===============
 
 To build your first program in CodeWorld, type (or just click on) this
-code:
+code
```
[the remark this answers is not in the document]
Comment (patrickscottd) [reply web/help/GuideUnit1.md:5]: I explained my reasoning yesterday. It's just how I write things and it's my own style. I'll fix it.

File: web/help/GuideUnit1.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (cdsmith) [reply web/help/GuideUnit1.md:1]: Let's leave it the way it is, then.  It's not that I have a strong opinion; just that I don't want the inconsistency, and changing everything throughout the guide (including the generated API documentation, etc.) is going to be a pain.

[the remark this answers is not in the document]
Comment (cdsmith) [reply web/help/GuideUnit1.md:1]: (I mean, the way it was, before this change, and still is in the rest of the guide)

**Comment by patrickscottd** [comment]
yes I understand

File: web/help/GuideUnit1.md (hunk 14 above)
Comment (patrickscottd) [reply web/help/GuideUnit1.md:801]: I'll think about a correct strategy for this. That's why it's good to have you here and that you are patient. I have to think about how to word that information. There is a subtlety, well more than that, but definitely a difference.

File: web/help/GuideUnit1.md [hunk 19]
```diff
@@ -1,8 +1,8 @@
-Getting started
```
Comment (patrickscottd) [inline web/help/GuideUnit1.md:1]: okay

File: web/help/GuideUnit1.md [hunk 20]
```diff
@@ -444,58 +463,61 @@ Function    | Domain             | Range
...
-just provide a little bit of information about what arguments they need,
-and what type of result they have.  Read the two colons (**`::`**) as
-"has the type", and the arrow (**`->`**) as meaning a *function* from one
-kind of thing to another.  So the first line says "`drawingOf` has the type:
-function from pictures to programs".
+These lines, called **type signatures**, don't *define* the functions, they
+just provide a little bit of information about what arguments they need
+and what type of results they have.  The two colons (**`::`**) should be read as
+"has the type", while the arrow (**`->`**) denotes a *function* from one
+kind of thing to another.  So, the first line saying "`drawingOf` has the type
```
[the remark this answers is not in the document]
Comment (patrickscottd) [reply web/help/GuideUnit1.md:508]: oh I get it now

File: web/help/GuideUnit1.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (patrickscottd) [reply web/help/GuideUnit1.md:5]: fixed

File: web/help/GuideUnit1.md (hunk 16 above)
[the remark this answers is not in the document]
Comment (patrickscottd) [reply web/help/GuideUnit1.md:50]: fixed

File: web/help/GuideUnit1.md (hunk 17 above)
[the remark this answers is not in the document]
Comment (patrickscottd) [reply web/help/GuideUnit1.md:52]: fixed

File: web/help/GuideUnit1.md [hunk 21]
```diff
@@ -444,58 +463,61 @@ Function    | Domain             | Range
...
 rectangle :: (Number, Number) -> Picture
 ~~~~~
 
-These lines, called **type signatures**, don't *define* the functions; they
-just provide a little bit of information about what arguments they need,
-and what type of result they have.  Read the two colons (**`::`**) as
-"has the type", and the arrow (**`->`**) as meaning a *function* from one
-kind of thing to another.  So the first line says "`drawingOf` has the type:
-function from pictures to programs".
+These lines, called **type signatures**, don't *define* the functions, they
```
[the remark this answers is not in the document]
Comment (cdsmith) [reply web/help/GuideUnit1.md]: Not fixed.

File: web/help/GuideUnit1.md (hunk 20 above)
[the remark this answers is not in the document]
Comment (cdsmith) [reply web/help/GuideUnit1.md]: Not fixed?

File: web/help/GuideUnit1.md [hunk 22]
```diff
@@ -444,58 +463,61 @@ Function    | Domain             | Range
...
+and what type of results they have.  The two colons (**`::`**) should be read as
+"has the type", while the arrow (**`->`**) denotes a *function* from one
+kind of thing to another.  So, the first line saying "`drawingOf` has the type
+"function from picture to program".
 
 You can explore all of the functions the computer already knows when you
-use CodeWorld, by pressing Shift-Space or Ctrl-Space on a blank line in the
+use CodeWorld by pressing Shift-Space or Ctrl-Space on a blank line in the
 CodeWorld editor.  Try it!  The list gives all the names your computer
-already knows, and there are a lot of them!  By typing the first few
```
[the remark this answers is not in the document]
Comment (cdsmith) [reply web/help/GuideUnit1.md:492]: Not fixed.

File: web/help/GuideUnit1.md [hunk 23]
```diff
@@ -504,9 +533,10 @@ more hints, expand the sections below for an example and explanation.
     solidCircle :: Number -> Picture
     ~~~~~
 
-    This type signature tells you that the only argument to `solidCircle`
-    is a number, and the result is a picture.  Here's an example of a
-    program that uses `solidCircle`.
+    This type signature tells you that the only argument of `solidCircle`
+    is a number and the result is a picture.
```
Comment (cdsmith) [inline web/help/GuideUnit1.md:534]: Needs either a comma before and, or (preferred) "and **that** the result is a picture."

File: web/help/GuideUnit1.md [hunk 24]
```diff
@@ -516,7 +539,7 @@ more hints, expand the sections below for an example and explanation.
     You might have noticed that even though the type signature tells you
     the input is a number, it doesn't tell you what that number means!
     Type signatures just tell you the *form* needed to use the function,
```
[the remark this answers is not in the document]
Comment (cdsmith) [reply web/help/GuideUnit1.md]: Not changed.

File: web/help/GuideUnit1.md [hunk 25]
```diff
@@ -516,7 +546,7 @@ more hints, expand the sections below for an example and explanation.
...
-    not the meaning.  But if you experiment, you may discover that the
+    not the meaning.  But, if you experiment, you may discover that the
```
Comment (cdsmith) [inline web/help/GuideUnit1.md:546]: I think this comma is too casual.  Maybe just remove "But" entirely?

File: web/help/GuideUnit1.md [hunk 26]
```diff
@@ -525,15 +548,16 @@ more hints, expand the sections below for an example and explanation.
...
+    arguments, both of them being numbers, and the result is a picture.  
+    
+Here's an example of a program that uses `solidRectangle`
 
     ~~~~~ . clickable
     program = drawingOf(pic)
     pic = solidRectangle(7, 3)
     ~~~~~
 
-    The first number is the width of the rectangle, and the second
```
[the remark this answers is not in the document]
Comment (cdsmith) [reply web/help/GuideUnit1.md]: Not fixed.

File: web/help/GuideUnit1.md [hunk 27]
```diff
@@ -542,16 +573,17 @@ more hints, expand the sections below for an example and explanation.
...
 
     ~~~~~ . clickable
     program = drawingOf(pic)
     pic = thickCircle(5, 1)
     ~~~~~
 
     Functions beginning with "thick" draw shapes with a thick line, and
-    the thickness of that line is the last argument to the function.  So
+    the thickness of that line is the last argument to the function.
     The first argument is the radius of the circle, and the second is
```
Comment (cdsmith) [inline web/help/GuideUnit1.md:584]: It isn't clear here that the second sentence is only talking about `thickCircle`.

File: web/help/GuideUnit1.md [hunk 28]
```diff
@@ -561,17 +593,18 @@ more hints, expand the sections below for an example and explanation.
     ~~~~~
 
     This time, the type signature tells you that `thickRectangle` needs
-    three arguments, a new record!  All three arguments are numbers, and
-    the result is a picture.  Here's an example of a program that uses
-    `thickRectangle`.
+    three arguments, a new record!  All three arguments are numbers and
```
Comment (cdsmith) [inline web/help/GuideUnit1.md:593]: comma is needed before and

File: web/help/GuideUnit1.md [hunk 29]
```diff
@@ -542,36 +566,38 @@ more hints, expand the sections below for an example and explanation.
...
+Here's an example of a program that uses 'thickRectangle`.
 
     ~~~~~ . clickable
     program = drawingOf(pic)
     pic = thickRectangle(8, 4, 1)
     ~~~~~
 
     The first two arguments are the width and height of the rectangle.
-    The third and final argument is the thickness of the line to draw it
+    The third and final argument is the thickness of the line we use to draw it
```
[the remark this answers is not in the document]
Comment (cdsmith) [reply web/help/GuideUnit1.md]: Not resolved.  I think you can just revert this change.

File: web/help/GuideUnit1.md [hunk 30]
```diff
@@ -581,7 +607,7 @@ more hints, expand the sections below for an example and explanation.
 
     This was a trick question: `codeWorldLogo` isn't a function at all!
     It still has a type signature, but there is no arrow, because it's
-    just a picture.  That means there are no parentheses after it, and
+    just a picture.  That means there are no parentheses after it and it has
```
[the remark this answers is not in the document]
Comment (cdsmith) [reply web/help/GuideUnit1.md]: Not fixed.

File: web/help/GuideUnit1.md (hunk 1 above)
Comment (cdsmith) [reply web/help/GuideUnit1.md]: Not resolved.

File: web/help/GuideUnit1.md [hunk 31]
```diff
@@ -627,7 +660,7 @@ Coloring
 The first transformation you will use is coloring.  The `colored` function
 changes the color of a picture. This function expects two arguments: the
 preimage, and a new color.  The colors of the preimage don't matter at
-all; only the shapes involved.  The result of the `colored` function is a new
+all - only the shapes involved.  The result of the `colored` function is a new
```
Comment (cdsmith) [inline web/help/GuideUnit1.md:660]: Use two dashes (like `--`) for a long dash.

File: web/help/GuideUnit1.md [hunk 32]
```diff
@@ -664,34 +697,38 @@ program, by asking and answering a question.
...
     ~~~~~
 
-    If you ran this, you see a black circle. `redWheel` is still
-    defined to be a red circle; but that doesn't change anything
-    because `program` is defined to be a drawing of `wheel`, which
-    is still black.
+    If you ran this, you would see a black circle. `redWheel` is still
+    defined to be a red circle, but that doesn't change anything. 
+    Why?
+    It is because `program` is defined to be a drawing of `wheel`, which
```
Comment (cdsmith) [inline web/help/GuideUnit1.md:700]: If you meant to start a new paragraph, you need a blank line.

File: web/help/GuideUnit1.md (hunk 5 above)
Comment (cdsmith) [reply web/help/GuideUnit1.md]: Not resolved.

File: web/help/GuideUnit1.md (hunk 7 above)
Comment (cdsmith) [reply web/help/GuideUnit1.md:679]: Not resolved.

File: web/help/GuideUnit1.md (hunk 8 above)
Comment (cdsmith) [reply web/help/GuideUnit1.md]: Not fixed.

**Comment by patrickscottd** [comment]
is this right?
